src/components/tabController/TabBarItem.tsx

```diff
@@ -8,11 +8,13 @@ import {Colors, Typography, Spacings} from '../../style';
 import Badge, {BadgeProps} from '../badge';
 import View from '../view';
 import TabBarContext from './TabBarContext';
+import Constants from '../../commons/Constants';
+import {TabControllerBarProps} from './TabBar';
```

Comment:
change to `import type {TabControllerBarProps} from './TabBar';`
It might have a cyclic require because TabBar imports TabBarItem and TabBarItem imports TabBar
Maybe just importing with `type` is ok